Example/SmileID/HomeViewModel.swift

```diff
@@ -18,7 +18,7 @@ class HomeViewModel: ObservableObject,
 
     init(config: Config) {
         partnerId = config.partnerId
-        SmileID.initialize(config: config, useSandbox: SmileID.useSandbox)
+        SmileID.initialize(config: config, useSandbox: true)
```

Comment:
Is hardcoding this to be true always also intentional?

Reply:
This is the sample app, so I was defaulting it to use Sandbox. This matches the exact setup on android

Reply:
Aaah, gotcha!